src/find_db.cpp

```diff
@@ -138,7 +138,7 @@ std::string FindDbRecord_t<TDb>::GetInstalledPathFile(Handle& handle)
             {
                 MIOPEN_LOG_I2("Iterating over find db directory " << root_path.string());
                 std::vector<fs::path> all_files;
-                for(const auto& kinder : fs::recursive_directory_iterator(root_path))
+                for(const auto& kinder : fs::directory_iterator(root_path))
```

Comment:
@JehandadKhan so are we sure we will not have file structure like:
```
root_path
│   README.md
│   ASIC_1_db    
│
└───ASIC_2
│   │   ASIC_2_var1_db 
│   │   ASIC_2_var2_db 
│   │
│   └───ASIC_2_va3
│       │   ASIC_2_var3_bran1_db
│       │   ASIC_2_var3_bran2_db
│       │   ...
│   
```

Reply:
That is correct.